oci/layout/oci_transport_test.go

```diff
@@ -13,6 +13,17 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func TestMoreThanOneImageError(t *testing.T) {
+	ref, tmpDir := refToTempOCI(t)
+	defer os.RemoveAll(tmpDir)
+
+	ociRef := ref.(ociReference)
+	ociRef.image = ""
```

Comment:
Thanks!

Rather than affecting `refToTempOCI` for everyone, and the type casts here, could the test create its own manifest? And because this is read-only, there isn’t really a need to use temporary directories; the test case data could be an independent subdirectory in the expected format (and potentially editable using standard OCI-processing tools) in a `fixtures` subdirectory, like `oci_src_test.go` uses a `fixtures/manifest` subdirectory for its tests.

And maybe, to be consistent with the rest of this file, name this `TestGetManifestDescriptor`, with a note that it tests against only one regression and the rest of the function is still untested.